.github/workflows/action.yml

```diff
@@ -0,0 +1,47 @@
+name: Github action to create EKS cluster using eksctl and install DOP on it
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install eksctl
+      run: |
+        curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+        sudo mv /tmp/eksctl /usr/local/bin
+        sleep 10
+        eksctl version
+
+    - name: Checkout code
+      uses: actions/checkout@master
+
+    - name: Run Shellcheck
+      uses: bewuethr/shellcheck-action@v2
+
+##     TODO:: There is no go-file in repo. Thatswhy we are
+##     commenting this lines for now. 
+#
+#     - name: Run golangci-lint
+#       uses: actions-contrib/golangci-lint@v1
+
+    - name: Configure AWS credentials from secrets
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+    
+    - name: Create EKS cluster using eksctl and Install kubera on cluster
+      run: |
+        chmod +x ./utils/verify_kubera_on_eks.sh
+        ./utils/verify_kubera_on_eks.sh
+      env:
+        DOCKER_USER: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_PASS: ${{ secrets.DOCKER_PASSWORD }}
+
+    - name: Cluster and kubera cleanup
+      run: |
+        chmod +x ./utils/cleanup.sh
```

Comment:
can we break this into uninstall DOP & run postflight checks
uninstall EKS

Reply:
We have only one command in postflight check @AmitKumarDas